doc/modules/cross_validation.rst

```diff
@@ -240,16 +240,18 @@ prediction that was obtained for that element when it was in the test set. Only
 cross-validation strategies that assign all elements to a test set exactly once
 can be used (otherwise, an exception is raised).
 
-These prediction can then be used to evaluate the classifier::
 
-  >>> from sklearn.model_selection import cross_val_predict
-  >>> predicted = cross_val_predict(clf, iris.data, iris.target, cv=10)
-  >>> metrics.accuracy_score(iris.target, predicted) # doctest: +ELLIPSIS
-  0.973...
+Note on inappropriate usage of cross_val_predict
+================================================
+The function :func:`cross_val_score` takes an average of over cross-validation
+folds and is grouped in different ways on computation in comparison to :func:`cross_val_predict`.
```

Comment:
I find this vague and unclear. What does "is grouped in different ways on computation" mean?

Reply:
I was reframing the words of this deleted note 
"- -Note that the result of this computation may be slightly different from those
 -obtained using :func:`cross_val_score` as the elements are grouped in different
 -ways." 
Is this not true too ?

Reply:
The previous wording was readable. The new wording is not.

Reply:
Thanks! for reviewing. Will make it readable 👍